cmd/kpromo/cmd/pr/pr.go

```diff
@@ -207,8 +207,10 @@ func runPromote(opts *promoteOptions) error {
 	}
 
 	// Clone k8s.io
-	// We might want to set options to pass for the clone, nothing for now
-	gitCloneOpts := &gogit.CloneOptions{}
+	gitCloneOpts := &gogit.CloneOptions{
+		// Set a minimal depth to prevent downloading the whole repository.
+		Depth: 1,
```

Comment:
The question is if we break something, but yeah let's give it a try.

Reply:
That's why I tagged you and @cpanato. 😅 But given that this PR changes only the `pr` command, I think we can't break that much. And even if we do break it, we would probably realise it very quickly.